state/aws/dynamodb/metadata.yaml

```diff
@@ -0,0 +1,83 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: v1
+type: state
+name: aws.dynamodb
+version: v1
+status: stable
+title: "AWS DynamoDB"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-state-stores/setup-dynamodb/
+capabilities:
+  - crud
+  - transactional
+  - etag
+  - ttl
+authenticationProfiles:
+  - title: "AccessKey and secretKey"
+    description: "Authenticate using accessKey and secretKey."
+    metadata:
+      - name: accessKey
+        type: string
+        required: false
+        sensitive: true
+        description: |
+          ID of the AWS account with appropriate permissions to SNS and SQS.
+          Can be secretKeyRef to use a secret reference.
+        example:  "AKIAIOSFODNN7EXAMPLE"
+        default: ""
+      - name: secretKey
+        type: string
+        required: false
+        sensitive: true
+        description: |
+          Secret for the AWS user. Can be secretKeyRef to use a secret 
+          reference.
+        example:  "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
+        default: ""
+metadata:
+  - name: table
+    required: true
+    description: |
+      The name of the DynamoDB table to use.
+    example: '"Contracts"'
+    type: string
+  - name: region
+    required: false
+    description: |
+      The AWS region to the instance. See this page for valid regions:
+      https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html. 
```

Comment:
Please add links as link refs. See for example: https://github.dev/dapr/components-contrib/blob/f125940490d146284b7608a526f5f79d811f973e/bindings/postgres/metadata.yaml#L33